ClientLib/src/main/java/com/o3dr/services/android/lib/coordinate/LatLongAlt.java

```diff
@@ -14,24 +15,34 @@
      * Stores the altitude in meters.
      */
     private double mAltitude;
+    private Frame mFrame;
+
+    public LatLongAlt() {
```

Comment:
@billbonney Same comment as with `LatLong`. I fail to see the utility of a default constructor.